README.md

```diff
@@ -22,46 +22,74 @@
 [![Postgres](https://img.shields.io/badge/postgres-%23316192.svg?style=postgres&logo=postgresql&logoColor=white)](https://www.postgresql.org/)
 [![Redis](https://img.shields.io/badge/redis-%23DD0031.svg?style=flat&logo=redis&logoColor=white)](https://redis.io/)
 
-# Dev Website
 
-https://dev.offender-categorisation.service.justice.gov.uk/
```

Comment:
I think it would be still useful to keep it in, maybe to add pre-prod/prod urls as well?